distribution/docs/src/main/resources/_contents/configuring-from-configuration-files-contents.adoc

```diff
@@ -592,7 +592,7 @@ cp /path/to/old/data/dir/* /path/to/new/data/dir/.
 .Set the SOLR_CATALOG_DATA_DIR
```

Comment:
Should we change `SOLR_CATALOG_DATA_DIR` to `solr.data.dir` since that's the name of the property?

We could also be consistent and use the same caption used above, i.e., `Update system.properties file`.